tests/console/sssd_389ds_functional.pm

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: sssd test with 389-ds as provider
+#
+# Set up 389-ds in container and run test cases below:
+# 1. nss_sss test: look up user identity with id: uid and gid
+# 2. pam_sss test: ssh login localhost as remote user.
+# 3. write permission test: change remote user password with passwd
+# 4. sssd-sudo test: Sudo run command as another remote user with sudoers rules defined in server
+# 5. offline test: shutdown server, run test cases above again
+#
+# Detailed testcases: https://bugzilla.suse.com/tr_show_case.cgi?case_id=1768710
+#
+# Maintainer: Tony Yuan <tyuan@suse.com>
+
+package sssd_389ds_functional;
+use base 'consoletest';
+use testapi;
+use strict;
+use warnings;
+use utils;
+use version_utils;
+use registration 'add_suseconnect_product';
+
+sub run {
+    #    my ($self) = @_;
```

Comment:
need remove?

Reply:
Keep it as note that the test can run in serial console.